src/MSBuild.Sdk.SqlProj/Sdk/Sdk.props

```diff
@@ -26,7 +26,7 @@
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <NoPackageAnalysis>True</NoPackageAnalysis>
-    <NuspecFile>$(MSBuildProjectName).nuspec</NuspecFile>
+    <NuspecFile Condition="Exists('$(MSBuildProjectName).nuspec')">$(MSBuildProjectName).nuspec"</NuspecFile>
```

Comment:
This removes the requirement for the nuspec file to exist allowing the msbuild sdk to generate the nuspec file